ofono/modem.go

```diff
@@ -390,7 +469,7 @@ func (modem *Modem) GetMMSContexts(preferredContext dbus.ObjectPath) (mmsContext
 	}
 
 	for _, context := range contexts {
-		if (context.isTypeInternet() && context.isActive() && context.hasMessageCenter()) || context.isTypeMMS() {
+		if (context.isTypeInternet() && context.hasMessageCenter()) || context.isTypeMMS() {
```

Comment:
On Fri, May 29, 2015 at 7:04 AM, alfonsosanchezbeato <
notifications@github.com> wrote:

> In ofono/modem.go
> <https://github.com/ubuntu-phonedations/nuntium/pull/20#discussion_r31312524>
> :
>
> > @@ -390,7 +469,7 @@ func (modem *Modem) GetMMSContexts(preferredContext dbus.ObjectPath) (mmsContext
> >  	}
> >
> >  	for _, context := range contexts {
> > -		if (context.isTypeInternet() && context.isActive() && context.hasMessageCenter()) || context.isTypeMMS() {
> > +		if (context.isTypeInternet() && context.hasMessageCenter()) || context.isTypeMMS() {
>
> In this concrete case I think the parenthesis add readability so I would
> prefer to keep them.
>

Right, but if you run gofmt it will most likely remove them.

>  —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubuntu-phonedations/nuntium/pull/20/files#r31312524>.
>